app/src/main/java/com/soen490chrysalis/papilio/view/CreateActivity.kt

```diff
@@ -102,27 +99,23 @@ class CreateActivity : AppCompatActivity()
                 // Callback is invoked after the user selects media items or closes the
                 // photo picker.
 
-                if (uris.size > 5)
-                {
+                if (uris.size > 5) {
```

Comment:
The formatting in this file does not seem right. All these changes undo our formatting rules